tasks/transform_conformance/exec_runner.snap.md

```diff
@@ -0,0 +1,88 @@
+* private/instance/exec.js
+* private/parenthesized-optional-member-call/exec.js
+* private/parenthesized-optional-member-call-with-transform/exec.js
+* private-loose/access-before-declaration/exec.js
+* private-loose/parenthesized-optional-member-call/exec.js
+* private-loose/parenthesized-optional-member-call-with-transform/exec.js
+* public/computed-toPrimitive/exec.js
+
+# babel-plugin-transform-private-methods (108/115)
+* accessors-loose/get-only-setter/exec.js
+* accessors-privateFieldsAsProperties/get-only-setter/exec.js
+* accessors-privateFieldsAsSymbols/get-only-setter/exec.js
+* static-accessors/get-only-setter/exec.js
+* static-accessors-loose/get-only-setter/exec.js
+* static-accessors-privateFieldsAsProperties/get-only-setter/exec.js
+* static-accessors-privateFieldsAsSymbols/get-only-setter/exec.js
+
+# babel-plugin-transform-private-property-in-object (5/10)
+* assumption-privateFieldsAsProperties/rhs-not-object/exec.js
+* assumption-privateFieldsAsSymbols/rhs-not-object/exec.js
+* private/rhs-not-object/exec.js
+* private-loose/rhs-not-object/exec.js
+* to-native-fields/rhs-not-object/exec.js
+
+# babel-plugin-transform-logical-assignment-operators (0/4)
+* logical-assignment/anonymous-functions-exec/exec.js
+* logical-assignment/arrow-functions-exec/exec.js
+* logical-assignment/named-functions-exec/exec.js
+* logical-assignment/null-coalescing/exec.js
+
+# babel-plugin-transform-dynamic-import (2/9)
+* commonjs/exec-interop/exec.js
+* commonjs/exec-interop-null/exec.js
+* commonjs/exec-interop-string/exec.js
+* commonjs/exec-template-literal/exec.js
+* commonjs/exec-to-primitive/exec.js
+* commonjs/exec-to-string-order/exec.js
+* commonjs/exec-transpiled-dep/exec.js
+
+# babel-plugin-transform-nullish-coalescing-operator (0/2)
+* assumption-noDocumentAll/runtime-semantics/exec.js
+* nullish-coalescing/runtime-semantics/exec.js
+
+# babel-plugin-transform-optional-chaining (10/15)
+* assumption-noDocumentAll/parenthesized-expression-member-call/exec.js
+* general/parenthesized-expression-member-call/exec.js
+* general/parenthesized-expression-member-call-loose/exec.js
+* general/parenthesized-member-call/exec.js
+* general/parenthesized-member-call-loose/exec.js
+
+# babel-plugin-transform-async-generator-functions (23/26)
+* yield-star/ecma262-pr-2819/exec.js
+* yield-star/throw-method-with-finally/exec.js
+* yield-star-polyfill-corejs3/throw-method-with-finally/exec.js
+
+# babel-plugin-transform-object-rest-spread (18/33)
+* assumption-objectRestNoSymbols/rest-ignore-symbols/exec.js
+* assumption-pureGetters/rest-remove-unused-excluded-keys/exec.js
+* assumption-pureGetters/spread-single-call/exec.js
+* assumption-setSpreadProperties/expression/exec.js
+* assumption-setSpreadProperties/no-object-assign-exec/exec.js
+* assumption-setSpreadProperties-with-useBuiltIns/expression/exec.js
+* assumption-setSpreadProperties-with-useBuiltIns/no-object-assign-exec/exec.js
+* object-rest/null-destructuring/exec.js
```

Comment:
I am using `bun` and `vitest` to run test, they have some little differences.

For code below:

```js
import {expect,test} from 'vitest';
test('exec', () => {
	expect(() => {
		var {...x} = null;
	}).toThrow(/1/);
	expect(() => {
		var {x:x,...y} = null;
	}).toThrow(/null/);
});
```

Vitest's error message is "Cannot destructure 'null' as it is null.". Bun is "Right side of assignment cannot be destructured".

Bun will fail in this case(there are 4 cases now totally). But I still chose bun because another choice is too slowly now...

Vitest version will cost several minutes to generate the result, bun only cost several seconds.

Reply:
Nice investigation, let's go for bun!